specification/compute/resource-manager/Microsoft.Compute/stable/2021-07-01/compute.json

```diff
@@ -3513,6 +3513,13 @@
             "type": "string",
             "description": "The name of the resource group."
           },
+          {
+            "name": "$filter",
+            "in": "query",
+            "required": false,
+            "type": "string",
+            "description": "The filter to apply to the operation. Allowed values are 'startswith(instanceView/statuses/code, 'PowerState') eq true', 'properties/latestModelApplied eq true', 'properties/latestModelApplied eq false'."
```

Comment:
As mentioned here https://msazure.visualstudio.com/One/_workitems/edit/12097662 . The filter param is different. 

GET https://management.azure.com/subscriptions/576f5e67-b043-4ec9-81e9-47c7e8692cc4/providers/Microsoft.compute/virtualMachines?api-version=2021-07-01&**$filter='​virtualMachineScaleSet/id'** eq '/subscriptions/576f5e67-b043-4ec9-81e9-47c7e8692cc4/resourceGroups/CUSTOMDATATESTCANARY/providers/Microsoft.Compute/virtualMachineScaleSets/customdataBicep2'

Reply:
Same for below

Reply:
I have changed description to "The filter to apply to the operation. Allowed value is '​virtualMachineScaleSet/id' eq VmssArmURI." if you think is not reasonable. please let me know.